### DISCUSSION TITLE
Reusing previously assigned domain

### DISCUSSION THREAD

**Discussion by psyfire25** [vercel/community#6818] [Help] [answered]
### Summary

Hey everyone,

Super noob here! 

I have been trying to learn next and using my portfolio as a learning project. Im wanting to use a previously assigned domain but even though i deleted the project from my vercel dashboard it will not let me reuse the previouse domain. Is this possible?

Any help would be great.

### Steps to Reproduce

create a project from my github

Delete said project cos im a noob and messed it up

created a new project from my new github repo

attempted to set the domain to the old one

fail with erro domain already assigned to vercel project.

**Comment by amyegan** [comment]
Hey @psyfire25. I can look into it if you're able to share the domain here

**Comment by psyfire25** [comment]
Hey Amy, 

This is the one the project is running on [jjj-ashen-delta.vercel.app](https://jjj-ashen-delta.vercel.app/)

I have previously created a project with the same name which was deployed on jjj.vercel.app and would love to be about to run the project on that.

If you can help at all that would be great

**Comment by psyfire25** [comment, marked as answer]
oh i do apologies but it is an alias for a different accout not mine. Thanks anyway.
